src/screens/Rari/RariInfo.js

```diff
@@ -0,0 +1,228 @@
+import t from 'translations/translate';
+
+import ContainerWithHeader from 'components/Layout/ContainerWithHeader';
+import { BaseText, MediumText } from 'components/Typography';
+import Insight from 'components/Insight/Insight';
+import InsightWithButton from 'components/InsightWithButton';
+import { Spacing } from 'components/Layout';
+import Spinner from 'components/Spinner';
+
+import { themedColors } from 'utils/themes';
+import { fontStyles } from 'utils/variables';
+import { getDeviceWidth, formatFiat } from 'utils/common';
+import { convertUSDToFiat } from 'utils/assets';
+
+import { fetchRariFundBalanceAction } from 'actions/rariActions';
+
+import { defaultFiatCurrency } from 'constants/assetsConstants';
+
+import type { RootReducerState, Dispatch } from 'reducers/rootReducer';
+import type { Rates } from 'models/Asset';
+
+
+type Props = {
+  fetchRariFundBalance: () => void,
+  isFetchingFundBalance: boolean,
+  rariFundBalance: number,
+  rariApy: number,
+  baseFiatCurrency: ?string,
+  rates: Rates,
+};
+
+const bannerImage = require('assets/images/rari_pattern.png');
+const rariLogo = require('assets/images/rari_logo.png');
+
+const screenWidth = getDeviceWidth();
+const bannerWidth = screenWidth - 40;
+
+const MainContainer = styled.View`
+  padding: 16px 20px;
+`;
+
+const Subtitle = styled(MediumText)`
+  color: ${themedColors.text};
+  ${fontStyles.big};
+`;
+
+const Paragraph = styled(BaseText)`
+  color: ${themedColors.secondaryText};
+  ${fontStyles.medium};
+`;
+
+const Row = styled.View`
+  flex-direction: row;
+`;
+
+const Card = styled.View`
+  background-color: ${themedColors.card};
+  padding: 8px 16px 16px;
+  border-radius: 6px;
+`;
+
+const Banner = styled(CachedImage)`
+  width: ${bannerWidth}px;
+  height: ${bannerWidth * (120 / 335)}px;
+`;
+
+// https://github.com/kfiroo/react-native-cached-image/issues/125
+const RariLogoWrapper = styled.View`
+  position: absolute;
+  top: ${bannerWidth * (12 / 335)}px;
+  right: ${bannerWidth * (12 / 335)}px;
+`;
+
+const RariLogo = styled(CachedImage)`
+  width: ${bannerWidth * (48 / 335)}px;
+  height: ${bannerWidth * (48 / 335)}px;
+`;
+
+const RariInfoScreen = ({
+  fetchRariFundBalance,
+  isFetchingFundBalance,
+  rariFundBalance,
+  rariApy,
+  baseFiatCurrency,
+  rates,
+}: Props) => {
+  useEffect(() => {
+    fetchRariFundBalance();
+  }, []);
+
+  const fiatCurrency = baseFiatCurrency || defaultFiatCurrency;
+
+  const renderParagraph = (subtitle, paragraph) => {
+    return (
+      <>
+        <Subtitle>{subtitle}</Subtitle>
+        <Spacing h={14} />
+        <Paragraph>{paragraph}</Paragraph>
+        <Spacing h={30} />
+      </>
+    );
+  };
+  return (
+    <ContainerWithHeader
+      inset={{ bottom: 'never' }}
+      headerProps={{
+        centerItems: [{ title: t('rariContent.title.infoScreen') }],
+      }}
+      putContentInScrollView
+    >
+      <MainContainer>
+        <View>
+          <Banner source={bannerImage} />
```

Comment:
The banner has a white frame in the dark mode. It's not on the designs, but on the other hand it looks intentional, so I'm not sure it's a bug.

![image](https://user-images.githubusercontent.com/25530572/97626154-bb1aa900-1a29-11eb-8503-8e41329f0fac.png)